src/Mouf/Html/Renderer/Twig/MoufTwigExtension.php

```diff
@@ -116,6 +128,12 @@ public function translate(array $args = array())
         return call_user_func_array('iMsgNoEdit', $args);
     }
 
+    public function translateFilter($text, array $args = array())
```

Comment:
given that the ``translate`` function would still require at least 1 argument, you would make this change directly in ``translate`` and use the same implementation for the function and the filter